### DISCUSSION TITLE
Wsl Ubuntu 20.04 - Upgrade Podman version 3.4.2--> v4.1.0

### DISCUSSION THREAD

**Discussion by Clement-GitHub** [containers/podman#14260] [Q&A]
Hello,

I use podman on WSL 2 (ubuntu 20.04)

I need help for upgrade my podman 3.4.2 to 4.1.0, could you help me ?

Thanks in advance :)

**Comment by Clement-GitHub** [comment]
No solution or idea ?

**Reply by afbjorklund** [reply]
You would have to do your own packaging, we have the same issue in minikube.

Upgrading Ubuntu from 20.04 to 22.04 would bring it to 3.4.4 - but not up to 3.4.7

We don't need podman v4, though.

Only using it for CRI-O, anyway...
